Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -107,75 +107,227 @@ For the first fetch, the integration will fetch incidents that were created 10 m
+
+### Configure Incident Mirroring When the Trigger Incident is ServiceNow  
+When the trigger incident is ServiceNow, you use the default **ServiceNow Create Ticket and Mirror**
+incident type. You can also use the default incoming and outgoing mappers or optionally create custom
+mappers.
+
+#### STEP 1 - Configure the ServiceNow v2 Integration Instance for Mirroring.
+1. Navigate to **Integrations** and search for **ServiceNow v2**.
+2. Click **Add instance**.
+3. Select **Fetches incidents**.
+4. Under **Classifier**, select ServiceNow Classifier.  
+    **Note:**  
+    You define either the **Classifier** or the **Incident type** (not both). It is recommended to define the **Classifier** and leave **Incident type** N/A to enable labeling custom incident types under the ServiceNow Classifier.
+
+5. Under **Mapper (incoming)**, for default mapping select ServiceNow - Incoming Mapper. For custom mapping, follow the instructions in STEP 2 and then select the custom mapper name.
+6. Under **Mapper (outgoing)**, for default mapping select ServiceNow - Outgoing Mapper. For
+custom mapping, follow the instructions in STEP 3 and then select the custom mapper name.
+
+    ![image](https://raw.githubusercontent.com/demisto/content-docs/5781a9024dc9f7e6418c82d7b7318f07d49fc863/docs/doc_imgs/integrations/snowv2-configuration-settings.png)
+
+7. Enter the connection parameters.  
+    - Confirm whether your organization uses basic authorization or OAuth authorization (most use basic) and enter the relevant authorization details.  
+    -  Leave **ServiceNow API Version** empty since ServiceNow typically automatically provides the appropriate version. 
+8. Under **The query to use when fetching incidents**, the default query is to filter for new incidents: stateNOT IN6,7, where 6= resolved incidents and 7= closed incidents. For a different query, ask your ServiceNow representative to provide the exact states and their numbers.
+9. Select **Get incident attachments** to retrieve attachments from ServiceNow incident creation in Cortex XSOAR.
+10. Select the **Incident Mirroring Direction**:  
+    - **In** - Mirrors changes on the ServiceNow ticket in to the Cortex XSOAR ticket.
+    - **Out** - Mirrors changes on the Cortex XSOAR ticket to the ServiceNow ticket.
+    - **Both** - Mirrors changes both in and out on both tickets.
+11. Set the Timestamp field to query as part of the mirroring flow. This defines the ticket_last_update - the epoch timestamp when the ServiceNow incident was last updated. The default is sys_updated_on.
+12. Enter the relevant **Comment Entry Tag**, **Work Note Entry Tag**, and **File Entry Tag** values.  
+These values are mapped to the **dbotMirrorTags** incident field in Cortex XSOAR, which defines how Cortex XSOAR handles comments when you tag them in the War Room.  
+**Note:**  
+These tags work only for mirroring comments from Cortex XSOAR to ServiceNow.
```

Comment:
```suggestion
These tags work only for mirroring comments, work notes and files from Cortex XSOAR to ServiceNow.
```

Reply:
Hi @Shellyber - thank you for taking the time to review! The file is updated based on your and @synth-humbert 's feedback. Please confirm and approve the review.